test/e2e/e2e_suite_test.go

```diff
@@ -176,8 +176,7 @@ func installOperator() error {
 	if err != nil {
 		return fmt.Errorf("applying operator YAMLs: %w", err)
 	}
-	ctx := context.Background()
-	return wait.Poll(waitInterval, foreverTimeout, func() (bool, error) {
+	return wait.PollUntilContextTimeout(context.Background(), waitInterval, foreverTimeout, false, func(ctx context.Context) (bool, error) {
```

Comment:
I fixed the below error:

```shell
test/e2e/e2e_suite_test.go:180:9: SA1019: wait.Poll is deprecated: This method does not return errors from context, use PollWithContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. (staticcheck)
        return wait.Poll(waitInterval, foreverTimeout, func() (bool, error) {
```

xref: https://github.com/kubernetes/kubernetes/pull/117143